spec/ruby/library/coverage/result_spec.rb

```diff
@@ -91,15 +91,31 @@
     Coverage.result.should_not include(@config_file)
   end
 
-  it 'returns the correct results when eval is used' do
-    Coverage.start
-    require @eval_code_file.chomp('.rb')
-    result = Coverage.result
+  ruby_version_is '3.1'...'3.2' do
+    it 'returns the correct results when eval is used' do
+      Coverage.start
+      require @eval_code_file.chomp('.rb')
+      result = Coverage.result
 
-    result.should == {
-        @eval_code_file => [
-            1, nil, 1, nil, 1, nil, nil, nil, nil, nil, 1
-        ]
-    }
+      result.should == {
+          @eval_code_file => [
+              1, nil, 1, nil, 1, nil, nil, nil, nil, nil, 1
+          ]
+      }
+    end
+  end
+
+  ruby_version_is '3.2' do
+    it 'returns the correct results when eval is used' do
```

Comment:
It's quite difficult to notice the difference here.
Could you make the specs clearer?
For example the spec description could say in 3.1 it does not include eval lines and in 3.2 it does

Reply:
Is this just a matter of changing the test names?

Reply:
Yes, mostly the test names.